src/styles/main.scss

```diff
@@ -0,0 +1,117 @@
+$mainTxtColor: #060b35;
+$secondaryTxtColor: #616070;
+$cardMainCol: #fff;
+$cardBorder: #f3f3f3;
+$cardBtnColor: #00acdc;
+$gap: 16px;
+$mainFont: "Roboto", sans-serif;
+
+@mixin base($color, $borderCol) {
+  background-color: $color;
+  border: 1px solid $borderCol;
+  border-radius: 5px;
+}
+
+@mixin font($weight, $size, $line: $size) {
+  font-size: $size;
+  line-height: $line;
+  font-weight: $weight;
+}
```

Comment:
maybe instead of using function you can set the font in some top level tag, like body

Reply:
due to many different font styles, even after adding it to a body I still use it around 4 times so I decided to leave it :)